assets/schema_input.json

```diff
@@ -19,6 +19,46 @@
                 "format": "file-path",
                 "pattern": "^\\S+\\.mlst\\.json(\\.gz)?$",
                 "errorMessage": "MLST JSON file from locidex report, cannot contain spaces and must have the extension: '.mlst.json' or '.mlst.json.gz'"
+            },
+            "metadata_1": {
+                "type": "string",
```

Comment:
Could you add `"meta": ["metadata_1"]` here. This keyword is used by IRIDA Next to indicate a column is metadata. It is also used within Nextflow/nf-core to set this particular value in the `meta` map: https://nf-co.re/docs/contributing/modules#what-is-the-meta-map

Reply:
Updated in a1f69e86065537799c16873a82bc4e013e7eb479